aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move

```diff
@@ -1,3 +1,5 @@
+/// DEPRECATED in favor of `aptos_framework::aggregator_v2`.
```

Comment:
If the transition to v2 is not trivial, you may want to point to some useful doc on how to do that.